images/scorecard-proxy/cmd/proxy/main.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+
+	proxy "github.com/operator-framework/operator-sdk/pkg/ansible/proxy"
```

Comment:
Suggest moving the proxy package to something non-ansible specific since we're now using it outside that context.

Reply:
I would prefer doing this in a separate PR, but agree with you, plan on doing this once this is merged?

Reply:
Yeah, I think this should be done in a PR after this PR gets merged.